packages/account-v2/src/containers/ManualForm/manual-form.tsx

```diff
@@ -43,15 +43,15 @@ export const ManualForm = ({
             {({ dirty, isSubmitting, isValid }) => (
                 <Form>
                     <div className='flex flex-col min-h-screen w-full'>
-                        <div className='flex flex-col gap-1200'>
+                        <div className='flex flex-col gap-24'>
                             <ManualFormInputs
                                 isExpiryDateRequired={isExpiryDateRequired}
                                 selectedDocument={selectedDocument}
                             />
                             <ManualFormDocumentUpload selectedDocument={selectedDocument} />
                             <ManualFormFooter />
                         </div>
-                        <div className='sticky bottom-50 flex justify-end gap-800 bg-vp px-400 py-800 border-t-solid-grey-2 bg-solid-slate-50 border-solid border-t-100'>
+                        <div className='sticky bottom-0 flex justify-end gap-16 px-8 py-16 border-t-solid-grey-2 bg-solid-slate-0 border-solid border-t-2'>
```

Comment:
we need to refactor the footers not to use any `position` property rather than using flex 
Using position messes up the height and we have to define the offsets to compensate

Reply:
We will do this later

Reply:
Please add a TODO so we dont miss this